activesupport/lib/active_support/testing/isolation.rb

```diff
@@ -25,38 +25,39 @@ def run
 
       module Forking
         def run_in_isolation(&blk)
-          read, write = IO.pipe
-          read.binmode
-          write.binmode
+          IO.pipe do |read, write|
```

Comment:
Although many lines _appear_ to have changed, enabling GitHub's "hide whitespace" feature shows that the only actual changes are to this line and the line adding the corresponding `end`.